WebKitBrowser/CMakeLists.txt

```diff
@@ -62,6 +62,10 @@ set(PLUGIN_WEBKITBROWSER_XHRCACHE "true" CACHE STRING "XHR Cache")
 set(PLUGIN_WEBKITBROWSER_EXTENSION_DIRECTORY "Extension" CACHE STRING "Directory to store extension libraries")
 set(PLUGIN_WEBKITBROWSER_LOCALSTORAGE "%persistentpath%" CACHE STRING "HTML5 local storage path")
 set(PLUGIN_WEBKITBROWSER_COOKIESTORAGE "%persistentpath%" CACHE STRING "Browser cookie storage path")
+set(PLUGIN_WEBKITBROWSER_INDEXEDDB_ENABLE "false" CACHE STRING "Enable IndexedDB (HTML5) database")
+set(PLUGIN_WEBKITBROWSER_INDEXEDDB_PATH "%persistentpath%" CACHE STRING "The directory where IndexedDB databases will be stored")
```

Comment:
Hi @VeithMetro , as we now have indexeddbpath as %persistentpath%, it needs to be substituted [(Ticket)](https://jira.rdkcentral.com/jira/projects/METROL/issues/METROL-876?filter=myopenissues). As it has been done for localstorage: https://github.com/WebPlatformForEmbedded/ThunderNanoServicesRDK/blob/master/WebKitBrowser/WebKitImplementation.cpp#L2365

Reply:
@modeveci should I also add an `#ifdef USE_EXACT_PATHS` for indexedDBPath like it is done in case of LocalStorage?

Reply:
@VeithMetro Yes please

Reply:
@modeveci Added it with my last commit 😄